index.html

```diff
@@ -2478,73 +2480,76 @@ <h3>Zero-Knowledge Proofs</h3>
         <p>
 A zero-knowledge proof is a cryptographic method where an entity can prove
 to another entity that they know a certain value without disclosing the
-actual value. A real world example is proving that an accredited university
+actual value. A real-world example is proving that an accredited university
 has granted a degree to you without revealing your identity or any other
 personally identifiable information contained on the degree.
         </p>
         <p>
-The key capabilities introduced by zero-knowledge proof mechanisms are:
+The key capabilities introduced by zero-knowledge proof mechanisms are the
+ability of a <a>holder</a> to:
         </p>
-        <ol>
+
+        <ul>
           <li>
-The ability of a <a>holder</a> to combine multiple <a>verifiable credentials</a>
-from multiple <a>issuers</a> into a single presentation without revealing
-<a>credential</a> or <a>subject</a> identifiers to the <a>verifier</a>. This
-makes it more difficult for the <a>verifier</a> to collude with any of the
-issuers regarding the issued <a>credentials</a>.
+Combine multiple <a>verifiable credentials</a> from multiple <a>issuers</a> into
+a single <a>verifiable presentation</a> without revealing
+<a>verifiable credential</a> or <a>subject</a> identifiers to the
+<a>verifier</a>. This makes it more difficult for the <a>verifier</a> to collude
+with any of the issuers regarding the issued <a>verifiable credentials</a>.
           </li>
           <li>
-The ability of a <a>holder</a> to selectively disclose the claims in a <a>credential</a>
-to a <a>verifier</a> without requiring the issuance of multiple atomic <a>credentials</a>.
-This allows a <a>holder</a> to provide a <a>verifier</a> with precisely the information
-they need and nothing more.
+Selectively disclose the claims in a <a>verifiable credential</a> to a
+<a>verifier</a> without requiring the issuance of multiple atomic
+<a>verifiable credentials</a>. This allows a <a>holder</a> to provide a
+<a>verifier</a> with precisely the information they need and nothing more.
           </li>
           <li>
-The ability of a <a>holder</a> to produce a derived <a>credential</a> that is formatted
-according to the <a>verifier</a>'s data schema rather than the <a>issuer</a>'s, without
-needing to involve the <a>issuer</a> after <a>credential</a> issuance. This provides
-a great deal of flexibility for <a>holders</a> to use the <a>credentials</a> they have
-been issued.
+Produce a derived <a>verifiable credential</a> that is formatted according to
+the <a>verifier's</a> data schema rather than the <a>issuer's</a>, without
+needing to involve the <a>issuer</a> after <a>verifiable credential</a>
+issuance. This provides a great deal of flexibility for <a>holders</a> to use
+the <a>verifiable credentials</a> they were issued with.
```

Comment:
"they were issued with." 
makes no sense to me.

The original language 
"they have been issued." 
or
"they were issued."
is more clear in my opinion.

Reply:
How about this:
 "... flexibility for \<a>holders\</a> to use their issued \<a>verifiable credentials\</a>."

Reply:
I think that works.